src/mmw/js/src/analyze/views.js

```diff
@@ -215,14 +187,6 @@ var TabPanelsView = Marionette.CollectionView.extend({
     },
     childView: TabPanelView,
 
-    events: {
-        'shown.bs.tab li a ': 'triggerBarChartRefresh'
-    },
-
-    triggerBarChartRefresh: function() {
-        $('#analyze-output-wrapper .bar-chart').trigger('bar-chart:refresh');
```

Comment:
I'm not really sure what this code was supposed to do. Deleting it has had no effect.